test_utils.go

```diff
@@ -365,6 +365,7 @@ func createTestPeer(notifier chainntnfs.ChainNotifier,
 			htlcswitch.DefaultFwdEventInterval),
 		LogEventTicker: ticker.New(
 			htlcswitch.DefaultLogInterval),
+		RejectHTLC: false,
```

Comment:
similar comment here re: not being needed

Reply:
fixed!